btx/diagnostics/run.py

```diff
@@ -429,6 +454,77 @@ def visualize_energy_stats(self, output=None, outlier_threshold=-np.inf):
             if output is not None:
                 fig.savefig(output, dpi=300, bbox_inches='tight')
     
+    def visualize_stats_interactive(self, y_key='mean', color_by='beam_energy_eV'):
+        """
+        Plot trajectories of run statistics, interactively.
+
+        Parameters
+        ----------
+        y_key : str
+            self.stats_final key to plot
+        color_by : str
+            self.stats_final key used to color
+        """
+        hv.extension('bokeh')
+
+        y = self.stats_final[y_key]
+
+        df = pd.DataFrame({
+            'evt_id' : np.arange(y.shape[0]), 
+            'y' : self.stats_final[y_key], 
+            'color_by' : self.stats_final[color_by]})
+        scatter = hv.Scatter(df, kdims=['evt_id'], vdims=['y', 'color_by'])
+        return scatter.opts(opts.Scatter(tools=['hover'])).opts(width=800, color_index='color_by')
```

Comment:
I still get a deprecation warning here:
```
WARNING:param.PointPlot: The `color_index` parameter is deprecated in favor of color style mapping, e.g. `color=dim('color')` or `line_color=dim('color')`
:Scatter   [evt_id]   (y,color_by)
```

Reply:
Hmmm weird. I'm going to ignore it for now since I'm not sure what's the issue.